source/principles/architecture/index.html.md.erb

```diff
@@ -34,10 +34,11 @@ Design for user needs identified by research, not for generic reuse - that is pr
 
 > Example: [Get School Experience](https://schoolexperience.education.gov.uk/) needed to link historical requests with new requests but did not want to provide user accounts, so they identified a technique for matching records used by the analysts and created a UI pattern to get the user to match the records. This technique was followed by Get into Teaching to match existing case records and has since been consistently copied by Claim additional payments for Teaching and Continuing Professional Development to enable matching against their own records and the DQT.
 
-## Stand on the shoulders of giants
-Our efforts should be put where we really add value. Focus on what differentiates your service from others, these are strategic  and make use of existing technologies where it facilitates your specific needs.  Focusing on what makes your service different is a strategic advantage, and using existing services.
+## Focus on creating value
+Our efforts should be put where we really add value. Focus on what differentiates your service from others, these are strategic and make use of existing technologies where it facilitates your specific needs. Focusing on what makes your service different is a strategic advantage, and using existing services.
+
+Do away with the “undifferentiated heavy lifting” of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor-lock in and unnecessary spending by researching and using open source software when appropriate.
```

Comment:
Feels like the double-quotes here makes us unsure of the name here, or at least I think we should use a slightly more assertive tone here. 

```suggestion
Do away with the undifferentiated heavy lifting of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor-lock in and unnecessary spending by researching and using open source software when appropriate.
```

Reply:
Also, I believe it should be "vendor lock-in".

```suggestion
Do away with the “undifferentiated heavy lifting” of managing infrastructure, i.e. provisioning and maintaining compute and other resources. Prevent vendor lock-in and unnecessary spending by researching and using open source software when appropriate.
```